test/resource.cpp

```diff
@@ -51,6 +51,67 @@ TEST(CampResource, GetPlatform)
   ASSERT_EQ(static_cast<const Resource>(Omp()).get_platform(), Platform::omp_target);
 #endif
 }
+TEST(CampResource, Compare)
+{
+  Resource h1{Host()};
+  Resource h2{Host()};
+  Host h; Resource h3{h};
+
+  ASSERT_TRUE(h1 == h1);
+  ASSERT_TRUE(h1 == h2);
+  ASSERT_TRUE(h3 == h);
```

Comment:
Please test both orders.
```suggestion
  ASSERT_TRUE(h3 == h);
  ASSERT_TRUE(h == h3);
```

Reply:
and both orders of !=